.config/nextest.toml

```diff
@@ -33,3 +33,7 @@ retries = 1
 [[profile.default.overrides]]
 filter = 'test(compile_fail)'
 slow-timeout = { period = "3m", terminate-after = 2 }
+
+[[profile.default.overrides]]
+filter = 'test(~oom_test)'
+threads-required = "num-test-threads"
```

Comment:
I think we can disable these by default by setting a `default-filter` https://nexte.st/docs/configuration/